bench/vibrato-bench/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use std::env;
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+
+use vibrato::{Dictionary, Tokenizer};
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+
+    let rootdir = env!("CARGO_MANIFEST_DIR");
+    let dictname = &args[1];
+
+    let reader = BufReader::new(
+        File::open(format!("{}/resources_{}/system.dic", rootdir, dictname)).unwrap(),
+    );
+    let dict = unsafe { Dictionary::read_unchecked(reader).unwrap() };
+    let mut tokenizer = Tokenizer::new(&dict);
```

Comment:
Not essential to this PR issue, but what is the reason Tokenizer is mutable?

If mutable is required, it becomes practical disadvantages, such as the inability to use the tokenizer from multiple threads when building a search index.

Reply:
`Tokenizer` reuses once-allocated memory for quickly constructing a lattice graph. The current version is designed for single-threaded use. Of course, extension to multi-threading would be an important future task.